src/routes/login/+page.svelte

```diff
@@ -15,15 +17,30 @@
   let fullError: string | null = null;
   let loginButtonText = 'Login';
   let password = '';
+  let reason = $page.url.searchParams.get('reason');
   let username = '';
   let usernameInput: HTMLInputElement | null = null;
 
+  const JWT_EXPIRED = 'JWTExpired';
```

Comment:
I'm wondering if we should export this `const` from the `utilities/permissions.ts` file and import it here.

Reply:
Yep, good idea.